apps/masterbots.ai/components/routes/chat/chat-search-input.tsx

```
@@ -36,50 +63,78 @@ export function ChatSearchInput({
   }
 
   React.useEffect(() => {
-    fetchSearchPlaceholder() // Updates search placeholder based on active category or chatbot.
+    fetchSearchPlaceholder()
   }, [chatbot, activeCategory])
 
   React.useEffect(() => {
```

Biome

[error] This hook does not specify all of its dependencies: searchInThread

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)